runtime/queries/smali/highlights.scm

```diff
@@ -0,0 +1,216 @@
+; Types
+
+(class_identifier
+  (identifier) @type)
+
+(primitive_type) @type.builtin
+
+((class_identifier
+   . (identifier) @_first @type.builtin
+   (identifier) @type.builtin)
+  (#any-of? @_first "android" "dalvik" "java" "kotlinx"))
+
+((class_identifier
+   . (identifier) @_first @type.builtin
+   . (identifier) @_second @type.builtin
+   (identifier) @type.builtin)
+  (#eq? @_first "com")
+  (#any-of? @_second "android" "google"))
```

Comment:
We don't currently support `#any-of?`. These will need to be rewritten to use `#match?` instead, for example `(#match? @_second "^(android|google)$")`